README.md

```diff
@@ -17,8 +17,13 @@ These files also provide
 
 ## Lozenge
 
-Pollen mode makes it easy to insert lozenge in a non-disturbing way:
-insert `@` and hit the tab. `@` will be turned into lozenge.
+Pollen mode makes it easy to insert lozenge. `@` will be turned into
+lozenge. To insert `@`, press `@` twice. The preceding lozenge will
+be turned into an `@`.
+
+For the old behavior to insert the lozenge, (press `@` and then `TAB`),
```

Comment:
Thanks for the consideration (even though I don't think the old behavior is a good design). `pollen-rock` is using `@` to insert the lozenge, so I am so glad you made this change!